docs/zh/tutorial/common architecture.md

```diff
@@ -0,0 +1,135 @@
+# 实时决策系统中 OpenMLDB 的常见架构整合方式
+
+OpenMLDB 提供了一个线上线下一致性的实时特征计算平台。对于如何在实际业务系统中整合 OpenMLDB，构建完整的机器学习平台，OpenMLDB 提供了灵活的支持。本文关注基于 OpenMLDB，在企业级业务系统中使用的常见架构。我们主要关注存储和计算两个方面：
+
+- 离在线数据存储架构：如何合理的进行离线和在线数据的存储，并维持离在线数据的一致性。
+- 实时决策应用架构：如何基于 OpenMLDB 的实时请求计算模式构建线上应用，包含常见的事中决策和实时查询应用架构。
+
+## 离在线数据的存储架构
+
+由于不同的性能和数据量的需求，在一般情况下，OpenMLDB 的离线和在线数据在物理上是分开存储：
+
+- **在线数据存储：** OpenMLDB 提供了一个高效的实时数据库（基于内存或者磁盘），主要目的为存储在线数据用于实时特征计算，而非全量数据。其主要特点为：
+  - 针对时序数据的毫秒级访问，默认基于内存
+  - 具备数据过期自动淘汰的能力（TTL），TTL 可以根据表格粒度进行设置，用于线数据库仅存放必需的时间窗口内的数据
+  - 默认基于内存的存储引擎虽然性能较高，但是可能存在内存消耗量较大的问题，可以在满足性能要求的前提下使用基于磁盘的存储引擎
+- **离线数仓**：OpenMLDB 本身并不提供独立的离线存储引擎，可以灵活支持不同的离线数仓和架构形式。
+
+以下讨论常见的离线和在线数据的存储架构。
+
+## 全量数据存储于实时数据库（不推荐）
+
+![实时决策1](C:\Users\User\Downloads\OpenMLDB\docs\zh\tutorial\images\实时决策1.png)
```

Comment:
1. use the english filename
2. change it to relative path, e.g. "images/figure1.png"

Reply:
apply those two changes to all figures accordingly

Reply:
also, there should be no space in the file name "common architecture.md" => "common_architecture.md"